scripts/update-snarkyjs-bindings.sh

```diff
@@ -12,11 +12,14 @@ WEB_BINDINGS="$SNARKY_JS_PATH/src/bindings/compiled/web_bindings"
 
 # 1. node build
 
-$DIR_PATH/build-snarkyjs-node.sh
+$DIR_PATH/build-snarkyjs-node-artifacts.sh
```

Comment:
? Why do you change this?

Reply:
the existing script failed to update the bindings when the TS compilation didn't work. but now some of our TS depends on the wasm-bindgen types. so there was sometimes a circular dependency were you could never compile. this change fixes it (compiling Rust bindings doesn't need TS to succeed)